lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -210,6 +213,10 @@ data class DeployAccountTransaction(
     @JsonNames("contract_class")
     val classHash: Felt,
 
+    // not in RPC spec
+    @SerialName("contract_address")
+    val contractAddress: Felt = Felt.ZERO,
+
```

Comment:
it is because of the gateway - in transaction version 1 this field is named differently. For now I have just used `JsonNames` for senderAddress field to work this around.

Reply:
Not sure to be honest, probably not